modules/tinymce/src/themes/silver/main/ts/ui/general/OuterContainer.ts

```diff
@@ -252,6 +252,13 @@ const partHeader = Composite.partType.optional<OuterContainerSketchDetail, Heade
   ]
 });
 
+const partSPAP = Composite.partType.optional<OuterContainerSketchDetail, HeaderSpec>({
+  name: 'spap',
```

Comment:
This should have a render function and be called like `renderHeader` above. This would also work well with what I suggested previously about moving the SPAP logic to a separate module.